### ISSUE TITLE
Install package of these repo with git UPM is nearly possible except there are some warning

### ISSUE THREAD

**Issue by Thaina** [google/play-unity-plugins#159]
I know this is not really officially support, but please allow this

This repo actually make a great layout and so I could actually install them with `?path=` parameter, which is UPM feature https://docs.unity3d.com/Manual/upm-git.html#subfolder

```json
{
  "dependencies": {
    "com.google.external-dependency-manager": "file:../TGZ/com.google.external-dependency-manager-1.2.167.tgz",
    "com.google.android.appbundle": "https://github.com/google/play-unity-plugins.git?path=/GooglePlayPlugins/com.google.android.appbundle",
    "com.google.play.assetdelivery": "https://github.com/google/play-unity-plugins.git?path=/GooglePlayPlugins/com.google.play.assetdelivery",
    "com.google.play.billing": "https://github.com/google/play-unity-plugins.git?path=/GooglePlayPlugins/com.google.play.billing",
    "com.google.play.common": "https://github.com/google/play-unity-plugins.git?path=/GooglePlayPlugins/com.google.play.common",
    "com.google.play.core": "https://github.com/google/play-unity-plugins.git?path=/GooglePlayPlugins/com.google.play.core",
    "com.google.play.instant": "https://github.com/google/play-unity-plugins.git?path=/GooglePlayPlugins/com.google.play.instant",
    "com.google.play.review": "https://github.com/google/play-unity-plugins.git?path=/GooglePlayPlugins/com.google.play.review",
  }
}
```

I have try it with these 7 packages but unfortunately there are 3 of them throw warnings

```
Script 'Packages/com.google.play.assetdelivery/Samples/AssetDeliveryDemo/** many files **.cs' will not be compiled because it exists outside the Assets folder and does not to belong to any assembly definition file.

Script 'Packages/com.google.play.billing/Runtime/Scripts/** many files **.cs' will not be compiled because it exists outside the Assets folder and does not to belong to any assembly definition file.

Script 'Packages/com.google.play.instant/Samples/SphereBlast/Scripts/** many files **.cs' will not be compiled because it exists outside the Assets folder and does not to belong to any assembly definition file.
```

![image](https://user-images.githubusercontent.com/1042507/142675773-c9839610-47ad-4545-ba78-44f58857a5af.png)

Especially `com.google.play.billing` which is the thing really required and throw this warning around `Runtime/Scripts` folder which seem to be really important

I wish I could install every package with this method because it make my project folder conveniently smaller and portable. Could you please consider fix these warning and support UPM git installation system like above?

Alternatively, I wish google would reopen google package registry

**Comment by jamcohen** [comment]
Hi @Thaina,

This seems like a pretty straightforward fix. I'll look into it.

Best,
Jake Cohen

**Comment by Thaina** [comment]
Thank you very much

**Comment by jamcohen** [comment]
Hey @Thaina,

Thanks again for raising this issue.

We resolved the issues with com.google.play.assetdelivery and com.google.play.instant.

However, we can't fix the com.google.play.billing package without breaking older versions. The billing package has different dependencies depending on the Unity version. Later versions of Unity depend on the UnityEngine.Purchasing assembly, but older versions do not.

If we added a .asmdef that included UnityEngine.Purchasing as an assembly reference, older versions would throw an error saying that assembly cannot be found. If we left out that reference, newer versions would throw an error saying they can't find the methods declared in UnityEngine.Purchasing.

Given that the billing package was deprecated in 1.6.0, I'm hesitant to spend time figuring out a solution to the above issue.

Thanks,
Jake
